tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Library             SeleniumLibrary
+Resource            ../../../Resources/Page/OCPLogin/OCPLogin.resource
+Resource            ../../../Resources/OCP.resource
+
+Test Setup          Dashboard Test Setup
+Test Teardown       Dashboard Test Teardown
+
+
+*** Variables ***
+${pachyderm_container_name}     Pachyderm
+${pachyderm_appname}            pachyderm
+
+
+*** Test Cases ***
+Verify Pachyderm Can Be Installed Using OpenShift Console
+    [Tags]    Tier2    ODS-1137
```

Comment:
Could you use the usual formatting for Tags?

```
[Tags]    Tier2
...         ODS-XYZ
```

lgtm other than this

Reply:
To fix in another PR